templates/system.json

```diff
@@ -35,11 +35,11 @@
     },
     "companycompanyassociation_delete": {
       "method": "DELETE",
-      "url": "crm/v4/objects/company/{{ entity.id }}/associations/company/{{ entity['toObjectId'] }}"
+      "url": "crm/v4/objects/contact/{{ entity['id'] }}/associations/company/{{ entity['toObjectId'] }}"
```

Comment:
This should be:

"url": "crm/v4/objects/company/{{ entity['id'] }}/associations/company/{{ entity['toObjectId'] }}" and not contact for the first one.